src/common/search.ts

```diff
@@ -0,0 +1,22 @@
+export type SearchSuggestionArgs = {
+  query: string;
+  version: number;
+  limit?: number;
+};
+
+export const defaultSearchLimit = 3;
+export const maxSearchLimit = 100;
+
+export const getSearchLimit = ({
```

Comment:
You can simply do `Math.max(Math.min(limit, maxSearchLimit), 1)`

Reply:
Yep, nice!